src/mpid/ch4/shm/ipc/src/ipc_p2p.h

```diff
@@ -26,13 +26,12 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_IPC_send_contig_lmt(const void *buf, MPI_Aint
                                                        MPI_Datatype datatype, int rank, int tag,
                                                        MPIR_Comm * comm, int context_offset,
                                                        MPIDI_av_entry_t * addr,
-                                                       MPIDI_SHM_IPC_type_t ipc_type,
+                                                       MPIDI_IPC_mem_attr_t attr,
                                                        MPIR_Request ** request)
 {
     int mpi_errno = MPI_SUCCESS;
     size_t data_sz;
-    MPI_Aint true_lb;
-    bool is_contig ATTRIBUTE((unused)) = 0;
+    bool is_contig = 0;
     MPIR_Request *sreq = NULL;
```

Comment:
`unused ` is needed when building with --enable-fast

Reply:
I will revert it